canvas_site_creator/templates/canvas_site_creator/index.html

```diff
@@ -81,6 +81,23 @@ <h3>
                             {% verbatim %}
                             {{ courseInstanceModel.totalCoursesWithoutCanvasSiteWithExternal }} external)
                             {% endverbatim %}
+                            {% comment %} <br>
+                            <button type="button" class="btn btn-link pull-right" id="createSitesButton" ng-disabled="createDisabled()" ng-click="handleCreate($event)" data-href="{% url 'canvas_site_creator:course_selection' %}">Continue</button> {% endcomment %}
+                        </li>
+                        <li>
+                            <ng-pluralize count="courseInstanceModel.totalCoursesWithoutCanvasSiteAndSyncToCanvasFalse"
```

Comment:
You mention "add an additional bullet" -- I don't see that in [Admin Console QA](https://canvas.qa.tlt.harvard.edu/accounts/11/external_tools/3), so I suppose this isn't deployed yet outside of a local install?